packages/core/services/management.js

```diff
@@ -0,0 +1,366 @@
+  dependencies: ['api'],
+  settings: {
+    settingsDataset: 'settings',
+    exportDir: './exports',
+    retainTmpExportsMs: 5 * 60 * 1000
+  },
+  async started() {
+    if (!this.createJob) {
+      this.logger.warn(
+        'The moleculer-bull scheduler is not available for the management service. Some feature might not work.'
+      );
+    }
+    this.broker.call('api.addRoute', {
+      route: {
+        name: 'management',
+        authentication: true,
+        path: '/.management/',
+        aliases: {
+          'POST /actor/:actorUri/export': 'management.exportActor',
+          'DELETE /actor/:actorUri': 'management.deleteActor'
+        }
+      }
+    });
+
+    if (!fs.existsSync(this.settings.exportDir)) fs.mkdirSync(this.settings.exportDir);
+  },
+  actions: {
+    deleteActor: {
+      params: {
+        actorUri: { type: 'string' },
+        iKnowWhatImDoing: { type: 'boolean', default: false, convert: true }
+      },
+      async handler(ctx) {
+        const { actorUri, iKnowWhatImDoing } = ctx.params;
+        const { webId } = ctx.meta;
+        if (!iKnowWhatImDoing) {
+          throw new Error(
+            'Please confirm that you know what you are doing and set the `iKnowWhatImDoing` parameter to `true`.'
+          );
+        }
+
+        if (!webId || (webId !== 'system' && webId !== actorUri)) {
+          throw403('You are not allowed to delete this actor.');
+        }
+
+        // Validate that the actor exists.
+        const actor = await ctx.call('auth.account.findByWebId', { webId: actorUri });
+        if (!actor) {
+          throw404('Actor not found.');
+        }
+        const dataset = actor.username;
+
+        // Delete account information settings data.
+        await this.broker.call('auth.account.setTombstone', { webId: actorUri });
+
+        // Delete uploads.
+        const uploadsPath = path.join('./uploads/', dataset);
+        await fs.promises.rm(uploadsPath, { recursive: true, force: true });
+
+        // Delete backups.
+        if (this.broker.registry.hasService('backup')) {
+          await this.broker.call('backup.deleteDataset', { iKnowWhatImDoing, dataset });
+        }
+
+        // Send `Delete` activity to the outside world (so they delete cached data and contact info, etc.).
+        const { outbox } = await ctx.call('ldp.resource.get', {
+          resourceUri: actorUri,
+          webId: 'system',
+          accept: MIME_TYPES.JSON
+        });
+        await ctx.call(
+          'activitypub.outbox.post',
+          {
+            '@context': 'https://www.w3.org/ns/activitystreams',
+            collectionUri: outbox,
+            type: ACTIVITY_TYPES.DELETE,
+            object: actorUri,
+            actor: actorUri,
+            // TODO: In the future, it would be good to send the activity to as many servers as possible (not just followers).
+            //  This is in order to delete cached versions of the account.
+            to: PUBLIC_URI
```

Comment:
I changed to `to: [followers, contacts, PUBLIC_URI]`
The followers collection will only be able to resolve after https://github.com/assemblee-virtuelle/semapps/issues/1271 is handled but it shouldn't cause trouble but only undeliverable activities (the contacts collection will be considered as actor which won't be able to be resolved correctly and fail).

Reply:
Currently only the "followers" collections can be directly addressed, see here:
https://github.com/assemblee-virtuelle/semapps/blob/master/src/middleware/packages/activitypub/services/activitypub/subservices/activity.js#L49-L59
As I wrote in Matrix, if we want to enable to address any collection (which is preferable than allowing `apods:contacts`, since this is very specific to ActivityPods), we will need to check the collection is owned by the sender (possibly with the `dc:creator` predicate).

Reply:
I don't understand your comment. Currently the Delete activity will not be sent to any contacts, so it is useless. Until the linked issue is solved, I would suggest to simply do a `activitypub.collection.get` and put all contacts in the list.

Reply:
Alright, I added an `activitypub.collection.get` for the contacts :)